cmd/certs.go

```diff
@@ -67,6 +71,19 @@ var certsCmd = &cobra.Command{
 		if err := utils.GenerateSSHKeyPair(filepath.Join(certsDir, "id_rsa"), filepath.Join(certsDir, "id_rsa.pub")); err != nil {
 			log.Fatal(err)
 		}
+		if ssid != "" && password != "" {
```

Comment:
What about cases with open ssid? (not that those are common nowadays, but still)

Reply:
In this PR we add support only for WiFiKeyScheme_WPAPSK. We can add WiFiKeyScheme_WPAEAP and WiFiKeyScheme_SchemeNOOP in next PRs.